### PR TITLE
updating eslint

### PR DESCRIPTION
**Pull request by OsamaAbdellateef**
# Description

Please include a summary of the change and which issue is fixed when no corresponding ticket is available. Add additional informations the reviewers should know.

[ ] This change requires a UI-Kit update - inform Alex Tirado

# What should be tested?

## Are they some special informations required to test something?

# Reviewers:

Which persons should review this PR? Add person with @mentions

### CODE REVIEW

File: config/eslint/index.js [hunk 1]
```diff
@@ -11,19 +17,33 @@ module.exports = {
     },
   },
   rules: {
-    // Disabled cause it doesn't currently work as of 01/02/2021
     'no-redeclare': 'off',
```
Comment (sebald) [inline config/eslint/index.js:20]: Why deleting the comments? Can we enable these rules again?

File: config/eslint/index.js [hunk 2]
```diff
@@ -11,19 +17,33 @@ module.exports = {
...
   },
   parserOptions: {
     babelOptions: {
       parserOpts: {
-        // Allow imports like `import pkg from './package.json' assert { type: 'json' };`
         plugins: ['importAssertions'],
       },
     },
   },
+  ignorePatterns: ['docs/.contentlayer/generated/ContentPage/_index.mjs'],
```
Comment (sebald) [inline config/eslint/index.js:32]: This should not be part of the shareable config, right? It's very specific to our use case

**Review by sebald — CHANGES_REQUESTED** [review]

**Comment by sebald** [comment]
@OsamaAbdellateef  conflicts! :)

File: config/eslint/index.js (hunk 2 above)
Comment (OsamaAbdellateef) [reply config/eslint/index.js:35]: I am so sorry, did notice or notified by this @sebald

Comment (sarahgm) [reply config/eslint/index.js:35]: @OsamaAbdellateef did you already resolve this?

Comment (aromko) [reply config/eslint/index.js:35]: @OsamaAbdellateef Is the change pushed? Didn't see it 😅

**Review by aromko — DISMISSED** [review]

File: config/eslint/index.js (hunk 2 above)
Comment (aromko) [reply config/eslint/index.js]: @OsamaAbdellateef Thanks for moving it into the global eslint. Now we merge it and should have an eye on it if all rules are applied.

**Pull request merged**